src/main/java/fr/nyvlem/graynaud/mcMMO/listeners/PlayerListener.java

```diff
@@ -0,0 +1,63 @@
+package fr.nyvlem.graynaud.mcMMO.listeners;
+
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerMoveEvent;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.scheduler.BukkitTask;
+import sun.applet.Main;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class PlayerListener implements Listener {
+
+    private static Plugin plugin;
+    public PlayerListener(Plugin plugin)
+    {
+        this.plugin = plugin;
```

Comment:
this.plugin est static, il faut pas utiliser this, fais : `PlayerListener.plugin = plugin;`